src/test/resources/no.nav.familie.ef.iverksett.cucumber/oppdrag/revurdering_opphørsdato_endringer_på_andeler.feature

```diff
@@ -0,0 +1,41 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Opphørsdato er den samme som tidligere - skal ikke sende opphørsdato på nytt når det finnes endringer senere i tiden
```

Comment:
Tar gjerne en prat for å skjønne denne selv.

Reply:
Det er startdato - kan sikkert endre "opphørsdato" til startdato i beskrivelsen

Reply:
Ser vi har et eget kort på å gjøre om alt av opphørsdato til startdato:
https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-8042
Det er en del flere steder opphørsdato er brukt. Jeg kan gjøre dette i egen PR :)